### DISCUSSION TITLE
Point Cloud Scene

### DISCUSSION THREAD

**Discussion by hybridherbst** [needle-tools/needle-engine-support#18] [Demo and Starter Ideas]
Point clouds look nice and can convey great moods. 
A point cloud sample that runs in AR and VR and has some nice audio and motion would be very nice.

Some inspiration:
- https://www.marinabaysands.com/museum/exhibitions/we-live-in-an-ocean-of-air.html
- https://twitter.com/objktcom/status/1530626460087631872
- etc.

**Comment by smakhtin** [comment]
@hybridherbst There is a nice 3d tiles implementation from NYT for Three.js https://github.com/nytimes/three-loader-3dtiles. Is there any documentation on how to create your own components?

**Reply by marwie** [reply]
hi, have a look at https://docs.needle.tools/scripting there you find examples and a lot of references on how to create your own components!

**Reply by marwie** [reply]
you can also check out https://engine.needle.tools/docs/scripting-examples.html and download the samples via https://engine.needle.tools/samples

**Reply by smakhtin** [reply]
Thx, will take a look.
